src/YouTrackSharp/AgileBoards/Backlog.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+
+namespace YouTrackSharp.AgileBoards
+{
+    public class Backlog
+    {
+        [JsonProperty("value")]
+        public string Value { get; set; }
+
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("visibleForGroup")]
+        public string VisibleForGroup { get; set; }
+
+        [JsonProperty("updatableByGroup")]
+        public object UpdatableByGroup { get; set; }
```

Comment:
Is `object` the most specific type here? Can this be made more specific?

Reply:
Did some research and changed it to string